lib/tests/streamlit/web/server/server_test.py

```diff
@@ -703,8 +706,6 @@ async def does_script_run_without_error(self):
         return True, "test_message"
 
     def setUp(self):
-        self._server = Server(None, None, "test command line")
-        self._server.does_script_run_without_error = self.does_script_run_without_error
```

Comment:
Passing a null ioloop instance as the first param to Server here breaks if another async test has already run, so this logic is now moved to `get_app`. (This is yet another "test execution order" fix)